src/card.tsx

```diff
@@ -578,7 +638,7 @@ export const MediaCard = React.forwardRef<HTMLDivElement, MediaCardProps>(
                         aria-label={isTouchable ? ariaLabel : undefined}
                     >
                         {isTouchable && <div className={styles.touchableMediaCardOverlay} />}
-                        <div className={styles.mediaCard}>
+                        <div className={styles.mediaCard} aria-hidden={isTouchable}>
```

Comment:
All cards except PosterCard were missing this, and it's important in order to avoid Safari from reading all the texts inside a link as separate links (same thing we did in rows)